src/components/pages/CaseOverview/EditCaseDetails.js

```diff
@@ -64,9 +52,15 @@ const EditCaseDetails = props => {
   const onEditSubmit = e => {
     e.preventDefault();
     axiosWithAuth()
-      .put(`/case/${caseId}`, formValues)
+      .put(`/cases/${caseId}`, formValues)
```

Comment:
Nice catch here, adjusting to the actual spelling for this endpoint! These can be tough to catch when you have endpoints with plural/non-plural words, so good eye.